moon/cfc_pain_sounds/sv_pain_relay.moon

```diff
@@ -1,4 +1,5 @@
 import Recipientfilter, IsValid, CurTime from _G
+math_min = math.min
```

Comment:
```suggestion
import min from math
```

I think `min` is descriptive enough to use without the library prefix, but I won't hold up the PR for it